src/_includes/base.njk

```diff
@@ -18,9 +20,15 @@
         {% if blog.title and blog.coverImage.file.url %}
             {% set ogTitle = blog.title %}
             {% set ogImage = "https:" + blog.coverImage.file.url %}
-            {% set ogUrl = "blogs/" + blog.title | slugify %}
+            {% set ogUrl = "https://ai.gov.uk/blogs/" + blog.title | slugify %}
```

Comment:
These links might cause issues on the preprod site

Reply:
Good point, but as this is only for sharing on social media it won't be an issue in reality